lib/parser.js

```diff
@@ -352,7 +359,7 @@ module.exports = function (args, opts, y18n) {
             }
           })
         } catch (ex) {
-          if (argv[configKey]) error = Error(__('Invalid JSON config file: %s', configPath))
+          if (argv[configKey]) error = Error(__('Invalid config file (%s): %s', ex.message, configPath))
```

Comment:
Good point -- I'm tempted to keep the default error; unless someone returns an error object -- and then we'll output that error (folks using the closure can perform their own locale work).